tour/content/welcome.article

```diff
@@ -54,11 +54,11 @@ El tour está disponible en otros idiomas:
 - [[https://go-tour-id2.appspot.com/][Indonesian — Bahasa Indonesia]]
 - [[https://go-tour-jp.appspot.com/][Japanese — 日本語]]
 - [[https://go-tour-ko.appspot.com/][Korean — 한국어]]
-- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
 - [[https://go-tour-pl1.appspot.com/][Polish — Polski]]
+- [[https://tour.go-zh.org/][Simplified Chinese — 中文（简体）]]
+- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
```

Comment:
## Description

1. It was organized based on the hyperlink text (i.e. the alphabetical order of the languages).
2. And it is put in English first, which is the original version, but if you want I can also put English in alphabetical order.

## Preview

![image](https://github.com/user-attachments/assets/dd9843e2-510f-4652-a45c-7b23b5709004)

Reply:
@ThiagoMowszet